DfciPkg/SettingsManager/SettingsManagerProvider.c

```diff
@@ -376,19 +372,23 @@ ProviderValueAsAscii (
         break;
       }
 
-      Value = AllocateZeroPool (ENABLED_STRING_SIZE);
+      if (v == ENABLE_INCONSISTENT) {
+        AsciiString = "Inconsistent";

```

Comment:
It would be ideal to have these string literals defined in a header file. Not necessary for your change though. Feel free to resolve this comment if you choose not to do this.

Reply:
Done. Defined strings in DfciSystemSettingStrings.h

Reply:
Thanks! This is exactly what I was looking for.